### PR TITLE
Fixed key portions of discover page

### PR DESCRIPTION
**Pull request by G-Arv**
Main features to fix are visible, but there's a few issues:
1. For some reason when I try to put the instagram/facebook icons into the HTML, the other half has issues render its side, so I've just specified verbally which tag is which. 
2. I made dropdown features based on specific criteria (rather than just clean). I wasn't too sure how to map the items given this, so the current map reads [id: wasClicked (t/f)]. I've included each of the ids in a querySelector tag - lmk how I should fix it.

### CODE REVIEW

File: src/client/data/DatabasePouchDB.js
Comment (ashleybhandari) [inline src/client/data/DatabasePouchDB.js:1]: uncomment so main doesn't get messed up (it's working for everyone else idk why not w you)

File: src/client/index.html
Comment (ashleybhandari) [inline src/client/index.html:1]: uncomment

**Comment by G-Arv** [comment]
I uncommented these items - lmk if there's any issues.

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 1]
```diff
@@ -157,10 +165,92 @@ export class DiscoverView {
...
+            //     "semester": false,
+            //     "month": false,
+            //     "half year": false,
+            //     "full year": false
+            // } // commented out for formatting
+        }
+
+        // Overarching filter
+        const filter = document.createElement("div");
+        filter.classList.add("discover-filter");
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:228]: Put styling in styes.css

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 2]
```diff
@@ -176,6 +266,88 @@ export class DiscoverView {
...
+            const leaseFullYearBox = elm.querySelector('#fullYearBox');
+
+
+            // initial map, given all elements are false, currently contains all ids
+            const filterMap = new Map([
+                // items from user
+                // items for gender
+                ['femaleBox', false], // gender.identity === female
+                ['maleBox', false], // gender.identity === male
+                ['nonbinaryBox', false], // gender.identity === nonbinary
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:305]: You can use the `checked` property to grab the values in the map itself. This way you can also get rid of the long list of declarations.
```
['femaleBox', elm.querySelector('#femaleBox').checked],
['maleBox', elm.querySelector('#maleBox').checked],
etc.
```

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:305]: Oh I was confused about that - lemme fix it

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 3]
```diff
@@ -157,10 +165,89 @@ export class DiscoverView {
...
+
+        // Dictionary to filter users (based on housing preferences)
+        const preferenceOptions = {
+            "Rent": {
+                "minimum rent": false,
+                "maximum rent": false
+            },
+            "Occupants": {
+                "minimum occupants": false,
+                "maximum occupants": false
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:216]: The numerical input ones are gonna be annoying to implement so could you do any other property haha sorry

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:216]: No worries, I wasn't too sure about those either - I suppose room type and building type could replace these

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 4]
```diff
@@ -157,10 +165,89 @@ export class DiscoverView {
...
+            },
+            "Noise": {
+                "low noise": false,
+                "medium noise": false,
+                "loud noise": false
+            },
+            "Guests": {
+                "no to less guests": false,
+                "some guests": false,
+                "any number of guests": false
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:199]: Could you change the slider values to their property names in User (so all lowercase) and the keys in the inner object to 1-3? it'll make it easier to implement.
```
'clean': {  '1': false, '2': false, '3': false }
```

File: src/client/scripts/views/SignedIn/DiscoverView.js (hunk 3 above)
Comment (ashleybhandari) [reply src/client/scripts/views/SignedIn/DiscoverView.js:216]: Yeah dw abt adding everything, just a few things. The application doesn't have to be completely polished, just enough to show a start.

**Comment by G-Arv** [comment]
It looks like if I change the slider values to 1-3, it'll make all of them have the same IDs - is there a different way to do it?

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 5]
```diff
@@ -50,20 +49,18 @@ export class DiscoverView {
...
         const curProfile = unseen[this.#unseenIndex];
 
-        // filter bar
-        // const filterBar = await this.#renderFilterBar();
-        // this.#discoverViewElm.appendChild(filterBar);
-
         // left side of page: pic, name; bio as well if user has housing
         const bioSection = this.#addBioSection();
         this.#discoverViewElm.appendChild(bioSection);
+        // profileSection.appendChild(bioSection);
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:55]: remove unnecessary comments

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 6]
```diff
@@ -192,6 +158,7 @@ export class DiscoverView {
     #addBioSection() {
         const elm = document.createElement('div');
         elm.classList.add('discover-bio');
+        elm.style.marginTop = "50px";
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:161]: styling goes in styles.css

File: src/client/styles.css [hunk 7]
```diff
@@ -1528,3 +1528,13 @@ body {
     grid-column: 1 / 3;
     text-align: center;
 }
+
+.discover-filter {
```
Comment (ashleybhandari) [inline src/client/styles.css:1532]: remove `.discover-filter` if it's not being used

File: src/client/scripts/views/SignedIn/DiscoverView.js [hunk 8]
```diff
@@ -331,6 +309,23 @@ export class DiscoverView {
         if (user.hasHousing && user.description) {
             container.querySelector('.bio-description').innerText = user.description;
         }
+
+        // Adds user social media
+        if(user.socials.ig !== "") {
```
Comment (ashleybhandari) [inline src/client/scripts/views/SignedIn/DiscoverView.js:314]: Doesn't work, try running

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:314]: That's a bit weird - I'm able to see the uncommented code on my end. Should I take out social media for now?

Comment (ashleybhandari) [reply src/client/scripts/views/SignedIn/DiscoverView.js:314]: My bad, I see it, it's messed up the layout though
image

Comment (ashleybhandari) [reply src/client/scripts/views/SignedIn/DiscoverView.js:308]: There's a margin-top for the discover-info class that's really big for some reason

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:308]: That was something I forgot to take out that was used for getting the old filter to render - I'll take it out

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:307]: I committed it - for some reason the error didn't show up on my end when testing initially. Is the layout the same now?

Comment (ashleybhandari) [reply src/client/scripts/views/SignedIn/DiscoverView.js:307]: It looks good, thank you. Feel free to merge.

Comment (G-Arv) [reply src/client/scripts/views/SignedIn/DiscoverView.js:307]: Sounds good - thanks for your help Ashley!

**Pull request merged**